### ISSUE TITLE
HowTo install

### ISSUE THREAD

**Issue by kmorber** [IKSG/nsf-file-server#36]
Hallo,

what to do with the domino update site?

BR

**Comment by jesse-gallagher** [comment]
Good point - I should add a bit in the documentation. For the time being, it's this process: https://ds-infolib.hcltechsw.com/ldd/ddwiki.nsf/xpAPIViewer.xsp?lookupName=XPages+Extensibility+API#action=openDocument&res_title=XPages_Extension_Library_Deployment&content=apicontent
